packages/code-du-travail-data/dataset/themes/index.js

```diff
@@ -90,7 +90,7 @@ async function main() {
 
 function extractSlug(str) {
   const [_, id, label] = str.match(/([0-9]+) - (.+)$/, "");
-  const [theme] = label.split("-").map(t => t.trim()).reverse();
+  const [theme] = label.split(" - ").map(t => t.trim()).reverse();
```

Comment:
Since the char a added by clement, perhaps we could use a dedicated char like 
`—`  or `•` or `»`  so we are sure it's our joining char